### PR TITLE
Fix the order matching

### PR DESCRIPTION
**Pull request by kienle371999**
In this PR, I fix the mechanism of orderbook to compute the right tokens based on the `clearing_volume`.

### CODE REVIEW

File: batcher/orderbook.mligo [hunk 1]
```diff
@@ -24,13 +25,89 @@ type match_calculation_type = EXACT_MATCH | LEFT_PARTIAL | RIGHT_PARTIAL
...
+      | EXACT -> 
+        let equivalent_amount = Float.div float_amount exchange_rate.rate in 
+        Math.get_rounded_number equivalent_amount
+      | PLUS ->
+        let equivalent_amount = Float.div (Float.mul float_amount constant_number) exchange_rate.rate in 
+        Math.get_rounded_number equivalent_amount
+
+
+(* 
+  The actual price when the sides are the same.
```
Comment (glottologist) [inline batcher/orderbook.mligo:66]: I don't understand what you mean by "when the sides are the same"?

Comment (kienle371999) [reply batcher/orderbook.mligo:66]: For example, the current exchange rate is `tzBTC/USDT` and the buy side is `tzBTC`.

Comment (glottologist) [reply batcher/orderbook.mligo:66]: Ah okay, I see what you are saying now.

**Comment by glottologist** [comment]
@kienle371999 this is not required now right?

**Pull request closed without merging**